### ISSUE TITLE
[cross_file] No example

### ISSUE THREAD

**Issue by stuartmorgan** [flutter/flutter#93073]
I was looking at Pana scores for our packages, and it's flagging that we don't have an example for `cross_file`. We definitely should have at least an example.md with some useful code samples, if not an example app.

/cc @ditman

**Comment by ditman** [comment]
Yes, this package also needs its web tests migrated to integration_test instead of flutter test --platform chrome, an example will eventually come naturally with that migration!

**Comment by ditman** [comment]
Related: https://github.com/flutter/flutter/issues/89693

**Comment by momshaddinury** [comment]
> Hi, I’d like to work on this issue. Can you please assign it to me? [@ditman](https://github.com/ditman)

Since the old PR is closed, I am going ahead and submitting a PR. This is my first time contributing. I'm looking for guidance if needed. 

Thanks.

**Comment by tirth-patel-nc** [comment]
@momshaddinury go ahead it's all yours, thanks for contributing 💙

**Comment by stuartmorgan-g** [comment]
@tirth-patel-nc and @momshaddinury please see my previous comment here; assigning issues to non-members has a poor track record (e.g., note that the previous person who asked for this issue to be assigned to them abandoned their PR without any follow-up).
